eos-tech-support/eos-usb-internet-writer

```diff
@@ -0,0 +1,229 @@
+    />
+</library>
+XML
+)
+
+run_kolibri() {
+    kolibri_home=$(realpath "$tmp_dir/workspace/.kolibri")
+    KOLIBRI_HOME="$kolibri_home" flatpak run \
+        --filesystem="$kolibri_home" \
+        --command=/app/libexec/kolibri \
+        org.learningequality.Kolibri "$@"
+}
+
+# Install required Flatpak apps
+apps_to_install=(
+    "com.endlessm.encyclopedia.en"
+    "org.learningequality.Kolibri"
+)
+echo "Installing apps ${apps_to_install[*]}. Make sure you have the Flatpak repo installed for those"
+for app in "${apps_to_install[@]}"; do
+    flatpak install -y --or-update "$app"
+done
+
+# Prepare tmp directory
+mkdir -p "$tmp_dir/workspace"
+mkdir -p "$tmp_dir/downloads"
+
+# Download Software packages
+wget -c -O "$tmp_dir/downloads/kiwix-win.zip" http://download.kiwix.org/release/kiwix-desktop/kiwix-desktop_windows_x64.zip
+wget -c -O "$tmp_dir/downloads/kiwix-macos.dmg" https://download.kiwix.org/release/kiwix-desktop-macos/kiwix-desktop-macos.dmg
+
+# TODO: use real URLs when LE have the official builds
+wget -c -O "$tmp_dir/downloads/kolibri-win.zip" http://eor-desktop:8000/kolibri-win.zip
+wget -c -O "$tmp_dir/downloads/kolibri-macos.zip" http://eor-desktop:8000/kolibri-macos.zip
```

Comment:
Can we use macros / constants for these URLs as well? This also helps if we need to update them in the future, especially if we move all constants to the beginning of the file.